src/shared/scrapers/PA/index.js

```diff
@@ -0,0 +1,299 @@
+// import datetime from '../../lib/datetime/index.js';
+import * as fetch from '../../lib/fetch/index.js';
+import maintainers from '../../lib/maintainers.js';
+// import * as parse from '../../lib/parse.js';
+import log from '../../lib/log.js';
+import provinceToIso2 from './province-to-iso2.json';
+
+function titleCase(string) {
+  let result = '';
+  for (let i = 0; i < string.length; i++) {
+    if (i === 0 || string.charAt(i - 1) === ' ') {
+      result += string.charAt(i).toUpperCase();
+    } else {
+      result += string.charAt(i).toLowerCase();
+    }
+  }
+  result = result.replace('Del', 'del'); // ugh.
+  return result;
+}
+
+const patientStatus = {
+  FALLECIDO: 'deaths',
+  ACTIVO: 'active',
+  RECUPERADO: 'recovered',
+  UNKNOWN: 'unknown'
+};
+
+//   'UBICACIÓN_DEL_PACIENTE': 'FALLECIDO', - location of patient
+//          Can be FALLECIDO (deceased), AISLAMIENTO DOMICILIARIO (Home quarantine),
+//          HOSPITALIZADO (hospitalized), RECUPERADO (recovered), UCI (intensive care)
+const patientLocation = {
+  FALLECIDO: 'deaths',
+  'AISLAMIENTO DOMICILIARIO': 'quarantine',
+  HOSPITALIZADO: 'hospitalized',
+  UCI: 'icu',
+  RECUPERADO: 'recovered',
+  UNKNOWN: 'unknown'
+};
+
+// Maybe this serves as a prototype for the schema we need to develop.
+const templateDataEntry = {
+  // country: null, // taken from scraper object.
+  state: null,
+  // fail src/events/crawler/scrape-data/run-scraper.js:16:13
+  // county: null,
+  // city: null,
+  cases: 0,
+  deaths: 0,
+  quarantine: 0,
+  hospitalized: 0,
+  icu: 0,
+  recovered: 0
+};
+
+function getCountsFromLocation(location) {
+  const o = JSON.parse(JSON.stringify(templateDataEntry)); // "deep copy". Fuck.
```

Comment:
Apparently in node you can do this now, haven't tried it yet but seems a lot nicer than going to a string and back.
```
import v8 from 'v8';

const cloneDeep = obj => v8.deserialize(v8.serialize(obj));
```

Reply:
Cool, I'll try it.

Reply:
Can't do this anyway because that template object is what makes this scraper return "zero" instead of "no data"